auth-web/src/store/modules/business.ts

```diff
@@ -40,72 +35,102 @@ export default class BusinessModule extends VuexModule {
     this.businesses = businesses
   }
 
-  @Action({ commit: 'setBusinesses', rawError: true })
```

Comment:
I took out the "auto-commit" so it doesn't overwrite `businesses` on error. I now call setBusinesses() explicitly on line 133.